# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pan/stupid-gcc4-strictness.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- pan-0.14.2.91/pan/base/msort.c.orig	2005-09-20 22:40:20.167293327 +0200
+++ pan-0.14.2.91/pan/base/msort.c	2005-09-20 22:42:45.465692264 +0200
@@ -65,14 +65,18 @@
 			if ((*cmp) (b1, b2) <= 0)
 			{
 				--n1;
-				*((unsigned long int *) tmp)++ =
-					*((unsigned long int *) b1)++;
+				*((unsigned long int *) tmp) =
+					*((unsigned long int *) b1);
+				tmp += sizeof(unsigned long int);
+				b1 += sizeof(unsigned long int);
 			}
 			else
 			{
 				--n2;
-				*((unsigned long int *) tmp)++ =
-					*((unsigned long int *) b2)++;
+				*((unsigned long int *) tmp) =
+					*((unsigned long int *) b2);
+				tmp += sizeof(unsigned long int);
+				b2 += sizeof(unsigned long int);
 			}
 		}
 	else
